### PR TITLE
addding l.l.bean

### PR DESCRIPTION
**Pull request by mneko**

### CODE REVIEW

**Comment by bhousel** [comment]
Thanks @mneko and welcome 👋 
Kind of surprised we didn't have L.L.Bean in the index already 😆

**Comment by mneko** [comment]
I was shocked too, but this might explain how much trouble I was having with the iD editor.

I'll be dropping in more chains as I remember them

File: data/brands/shop/outdoor.json [hunk 1]
```diff
@@ -144,6 +144,21 @@
         "shop": "outdoor"
       }
     },
+    {
+      "displayName": "L.L.Bean",
+      "id": "llbean-783431",
+      "locationSet": {
+        "include": ["jp", "us"]
```
Comment (tas50) [inline data/brands/shop/outdoor.json:151]: They're in CA too

**Comment by bhousel** [comment]
> I was shocked too, but this might explain how much trouble I was having with the iD editor.
> I'll be dropping in more chains as I remember them

Thanks! We're happy to add any chains that are missing..
http://nsi.guide is a good resource to check what we have.
iD is a little bit behind what we have, but we will catch it up soon.

**Pull request merged**